cinema/views.py

```diff
@@ -69,7 +75,7 @@
 ):
     queryset = Movie.objects.prefetch_related("genres", "actors")
     serializer_class = MovieSerializer
-    authentication_classes = (TokenAuthentication,)
+    # authentication_classes = (TokenAuthentication,)
```

Comment:
The `authentication_classes` are commented out. Ensure this is intentional, as it might leave the endpoints unprotected. Consider enabling them for production to secure the API.